src/hotspot/share/gc/shenandoah/shenandoahGeneration.cpp

```diff
@@ -979,15 +978,3 @@ void ShenandoahGeneration::record_success_degenerated() {
   heuristics()->record_success_degenerated();
   ShenandoahHeap::heap()->shenandoah_policy()->record_success_degenerated();
 }
-
```

Comment:
I gather that this code was redundant/not needed.  I'm not sure removal of this code was highlighted in the commit comment.  Maybe we should also mention some code cleanup regarding generation collection times.

Reply:
Added this cleanup to the PR description. Yes, this code was a vestige of the old mmu based resizer.